Tests/UITests/TestApp/TestAppDelegate.swift

```diff
@@ -17,7 +17,9 @@ class TestAppDelegate: SpeziAppDelegate {
             AccessGuardModule(
                 [
                     .biometrics(identifier: "TestBiometricsIdentifier"),
-                    .code(identifier: "TestIdentifier", timeout: 10),
+                    .code(identifier: "TestIdentifier",
+                          codeOptions: [.all],
+                          timeout: 10),
```

Comment:
Generally recommend the following multi-line formatting: 
```suggestion
                    .code(
                        identifier: "TestIdentifier",
                        codeOptions: [.all],
                        timeout: 10
                    ),
```